libs/template/templates/default-python/template/{{.project_name}}/src/{{.project_name}}/common.py

```diff
@@ -0,0 +1,105 @@
+import pathlib
+import sys
+from abc import ABC, abstractmethod
+from argparse import ArgumentParser
+from typing import Any, Dict
+
+import yaml
+from pyspark.sql import SparkSession
+
+
+def _get_dbutils(
+    spark: SparkSession,
+):  # please note that this function is used in mocking by its name
+    try:
+        from pyspark.dbutils import DBUtils  # noqa
+
+        if "dbutils" not in locals():
+            utils = DBUtils(spark)
+            return utils
+        else:
+            return locals().get("dbutils")
+    except ImportError:
+        return None
+
+
+class Task(ABC):
```

Comment:
This is from dbx; I'm not entirely sure we want to keep it, but it kind of works for the very first template.

Reply:
We should check if this code also works with DB Connect out of the box. 

Otherwise, I like that we just copied the dbx example. I could imagine having and even simpler non-OOP python templae.